<launch>
    <node pkg="domino_vision_pkg" type="hand_detection.py" name="hand_detection"/>
</launch> 